docs/extensibility/form-section.md

```diff
@@ -205,7 +205,7 @@ Text widgets render a field as a text field. They are used by default for all st
 - **enum[]** - an array of options to generate an input field with a dropdown. Optionally can be a string containing a [JSONata](jsonata.md) expression returning an array of options.
 - **placeholder** - specifies a short hint about the input field value.
 - **required** - a boolean which specifies if a field is required. The default value is taken from CustomResourceDefintion (CRD); if it doesn't exist in the CRD, then it defaults to `false`.
-- **inputInfo** - a string below the input field that shows how to fill in the input.
+- **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```